lib/internal/source_map/prepare_stack_trace.js

```diff
@@ -139,17 +139,21 @@ function getErrorSource(
   originalLine,
   originalColumn
 ) {
-  let exceptionLine = '';
   const originalSourcePathNoScheme =
     StringPrototypeStartsWith(originalSourcePath, 'file://') ?
       fileURLToPath(originalSourcePath) : originalSourcePath;
   const source = getOriginalSource(
     sourceMap.payload,
     originalSourcePath
   );
+  if (typeof source !== 'string') {
+    return;
+  }
```

Comment:
```suggestion
  if (typeof source !== 'string') return;
  ```

Reply:
I'm not a fan of this one-lined style. I'd be reluctant to change the style if there are no style guides or enforced lint rules.